index.js

```diff
@@ -4,11 +4,79 @@ const graphqlHTTP = require('express-graphql');
 const cors = require('cors');
 const root = require('./src/resolvers/RootResolver');
 const schema = require('./src/schema/schema');
+const GitHubStrategy = require('passport-github2').Strategy;
+const passport = require('passport');
+const partials = require('express-partials');
```

Comment:
I believe partials are required for graphql to render properly. I get the errors ```
localhost/:1 Uncaught (in promise) TypeError: Failed to fetch ``` without them. From the NPM docs --> Express-partials Adds partial rendering support to Express so that you can render multiple templates in a single render call. Each template is provided its own template data object on render and the method, by default, will respond to the request with a json object containing the partial name and associated rendered html.